src/composables/secureStorage.ts

```diff
@@ -0,0 +1,48 @@
+import { computed, ref, watch } from 'vue';
+import { IS_MOBILE_APP } from '@/constants';
+import { IWalletStorage } from '@/lib/WalletStorage';
+import { SecureMobileStorage } from '@/lib/SecureMobileStorage';
+import { EncryptedStorage } from '@/lib/EncryptedStorage';
+import type { IKey } from '@/utils';
+import { createCustomScopedComposable } from './composablesHelpers';
+import { useAccounts } from './accounts';
+
+export const useSecureStorage = createCustomScopedComposable(() => {
+  let composableInitialized = false;
+  const secureStorage = ref<IWalletStorage>();
```

Comment:
I think this doesn't need to be a reference (eactive state).

Reply:
I think it makes sense for it to be a reference since we are watching it on the `storageRef.ts` composable and we want certain things to happen every time its value is changed